src/BUILD

```diff
@@ -123,9 +123,8 @@ JAVA_TOOLS = [
     "//src/java_tools/buildjar/java/com/google/devtools/build/buildjar/jarhelper:srcs",
     "//src/tools/singlejar:embedded_tools",
     "//src/java_tools/import_deps_checker/java/com/google/devtools/build/importdeps:embedded_tools",
-    "//third_party/ijar",
+    # TODO(#6316): Move this target to the remote java tools archive.
```

Comment:
Is this only used by Java? I thought it's also used by `py_*` at the very least.

Reply:
Do you mean `//third_party/ijar:zipper`? I'm not sure, I plan to look into it after this PR is merged.

Reply:
Yep. Not very important for the time being, but something to keep in mind. I hope we can about Python depending on Java or vice versa, but it would require at least a bit of thinking.